File: aviary-api/src/main/kotlin/eu/bluehawkqs/aviary/api/models/Tournament.kt
package eu.bluehawkqs.aviary.api.models

import java.time.LocalDate

data class Tournament(
    val id: Int = 0,
    val name: String,
    val description: String,
    val location: String,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val registrationDeadline: LocalDate,
    val size: Int,
    val currentUserAttending: Boolean)

File: aviary-angular/src/app/models/Tournament.ts
export class Tournament {
  constructor(public id: number = 0,
              public name: string,
              public description: string,
              public location: string,
              public startDate: Date,
              public endDate: Date,
              public registrationDeadline: Date,
              public size: number,
              public currentUserAttending: boolean) {
  }
}
